tests-performance/src/binary-operations.cc

```diff
@@ -75,18 +18,14 @@ int main(int argc, char *argv[])
 
     Nfa lhs;
     Nfa rhs;
-    Mata::StringToSymbolMap lhs_stsm;
-    Mata::StringToSymbolMap rhs_stsm;
-    if (load_automaton(lhs_filename, lhs, lhs_stsm, "lhs") != EXIT_SUCCESS) {
+    Mata::OnTheFlyAlphabet alphabet;
+    if (load_automaton(lhs_filename, lhs, alphabet, SKIP_MINTERMIZATION, "lhs") != EXIT_SUCCESS) {
         return EXIT_FAILURE;
     }
-    if (load_automaton(rhs_filename, rhs, rhs_stsm, "rhs") != EXIT_SUCCESS) {
+    if (load_automaton(rhs_filename, rhs, alphabet, SKIP_MINTERMIZATION, "rhs") != EXIT_SUCCESS) {
```

Comment:
What is this SKIP_MINTERMIZATION thing?

Reply:
It is a configuration constant defined in each perf test source file. If manually set to true, mintermization of the loaded automaton will be skipped, otherwise, the loaded automaton will be mintermized before returning from the `load_automaton()` function.

Reply:
ok